test/Controller/AboutTechnologyControllerTest.php

```diff
@@ -14,7 +14,7 @@ public function it_displays_the_innovation_page()
         $crawler = $client->request('GET', $this->getUrl());
 
         $this->assertSame(200, $client->getResponse()->getStatusCode());
-        $this->assertSame('Innovation', $crawler->filter('.content-header__title')->text());
```

Comment:
Need to update the method name too.

Reply:
This file does not exist anymore

Reply:
It's `it_displays_the_innovation_page`, needs to be `it_displays_the_technology_page`.